recipes/ntplib/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ntplib" %}
+{% set version = "0.3.3" %}
+{% set sha256 = "c4621b64d50be9461d9bd9a71ba0b4af06fbbf818bbd483752d95c1a4e273ede" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed  .
+  skip: True  # [win]
```

Comment:
Why are we skipping windows here?
According to the docs this could be noarch: python maybe?